### PR TITLE
CSP - IPv4 Regex update

### PR DESCRIPTION
**Pull request by Ni-Knight**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-9363

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Base/ReleaseNotes/1_33_16.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#### Scripts
+
+##### CommonServerPython
+
+- Fixed an issue with the IPv4 regex in Common Server Python not supporting IPv4s with ports.
```
Comment (AradCarmi) [inline Packs/Base/ReleaseNotes/1_33_16.md:5]:
```suggestion
- Added support for IPv4s with ports in the IPv4s regex.
```

File: Packs/Base/ReleaseNotes/1_33_16.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+- Fixed an issue with the URL regex not capturing a hyphen in the URL path.
```
Comment (AradCarmi) [inline Packs/Base/ReleaseNotes/1_33_16.md:6]:
```suggestion
- Fixed an issue where the URL regex not capturing a hyphen in the URL path.
```

**Review by AradCarmi — CHANGES_REQUESTED** [review]
Well done Dror!

**Review by AradCarmi — APPROVED** [review]

File: Packs/AutoFocus/Integrations/FeedAutofocus/FeedAutofocus_test.py [hunk 3]
```diff
@@ -19,7 +19,7 @@
 TYPES = [
     "File",
     "IP",
-    "URL",
+    "IP",
```
Comment (MLainer1) [inline Packs/AutoFocus/Integrations/FeedAutofocus/FeedAutofocus_test.py:22]: Was it intended?

Comment (Ni-Knight) [reply Packs/AutoFocus/Integrations/FeedAutofocus/FeedAutofocus_test.py:22]: Yep, its an IP.

**Pull request merged**